pub/index.html

```diff
@@ -1,221 +1,154 @@
-                      
-                          <address>Written by Bunny Lebowski</address>
-                     
-                      
-                          <p>Lebowski ipsum i want a fucking lawyer, man. I want Bill Kunstler. Dolor sit amet, consectetur adipiscing elit praesent ac magna justo pellentesque ac lectus quis. Dude, please!… Is this your homework, Larry? Elit blandit fringilla a ut turpis praesent felis ligula, malesuada suscipit malesuada. What do you mean "brought it bowling"? I didn't rent it shoes. I'm not buying it a fucking beer. He's not gonna take your fucking turn, Dude. Non, ultrices non urna.</p>
-
-<p>I like your style, Dude. Sed orci ipsum, placerat id condimentum rutrum, rhoncus ac lorem aliquam. You thought, hey, a deadbeat, a loser, someone the square community won't give a shit about. Placerat posuere neque, at. They finally did it. They killed my fucking car. Dignissim magna ullamcorper in aliquam sagittis massa ac tortor ultrices faucibus curabitur. I'm as Jewish as fucking Tevye. Eu mi sapien, ut ultricies ipsum morbi eget risus nulla nullam vel nisi enim.</p>
-
-<p>…which would place him high in the runnin' for laziest worldwide—but sometimes there's a man… sometimes there's a man. Vel auctor ante morbi id urna vel felis lacinia placerat vestibulum turpis nulla, viverra. Jeez. I miss vinyl. Nec volutpat ac, ornare id lectus cras pharetra faucibus. What's a pederast, Walter? Tristique nullam non accumsan justo nulla facilisi. Hey, man, if my fucking ex-wife asked me to take care of her fucking dog while she and her boyfriend went to Honolulu, I'd tell her to go fuck herself. Integer interdum elementum nulla, nec eleifend nisl euismod.</p>
-
-<figure>
-<img src="lebowski3.jpg"></img>
-
-</figure>
-                      </article>
-                  </section>
-                  
-                <!-- .page-column-main -->
-            </body>     
-             <!-- .page-columns -->
-            <footer id="page-footer">
-                <div id="footerlogo">
-                    <a href="http://orlando.theironyard.com"><img src="bowling_ball.jpg"></a>
-                </div>
-                <div id="footercontent">
-                    <p>Lebowski | Orlando</p>
-                    <p>The Dude Abides</p>
-                </div>
-                
-            </footer>
-        
-
-        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
-        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')</script>
+<!--Third time's a charm-->
+
+        <header>
+            <h1>Vertical Timeline <small>v</small>3.0</h1>
+
+        <div class="nav">
+            <a href="#"><i class="fa fa-arrow-left"></i></a>
+            <a href="#"><i class="fa fa-arrow-right"></i></a>
+            <a href="#"><i class="fa fa-home"></i></a>
+            <a href="#"><i class="fa fa-envelope"></i></a>
+        </div>
+
+        </header>
```

Comment:
FYI, these headers are completely optional. I care more about the content below. Good work, though! Could you do the same with Unicode?